### ISSUE TITLE
bart minimal example bugs

### ISSUE THREAD

**Issue by WeiliangGuo** [ThilinaRajapakse/simpletransformers#379]
eval_data=eval_df wasn't given in the minimal bart example when "evaluate_during_training": True.

"save_eval_checkpoints": False,
"save_model_every_epoch": False,
but still lots of checkpoint files were saved into the default outputs folder. For me I just need the best model saved only.

Maybe it's better to add more default evaluation info since I can only see the eval loss values with current default setting.

Can you offer a link for downloading usable pretrained mbart model files?

Thank you :)

**Comment by ThilinaRajapakse** [comment]
The checkpoint folders are created, but the model itself is not saved.

The default metric is eval loss because the metrics for sequence to sequence models aren't straight forward. You can use custom metrics as necessary though.

You can find the pre-trained model [here](https://huggingface.co/transformers/pretrained_models.html).

**Comment by WeiliangGuo** [comment]
The files saved in the default output folder are huge in size.

I know where to download the pre-trained models, however the multilingual-bart is not available?

**Comment by abedkhooli** [comment]
So, is mbart supported or planned to be? Would like to explore facebook/mbart-large-cc25 for translation (fine-tuning).
